README.md

```diff
@@ -156,3 +156,22 @@ To list all installed versions of the module available for import:
 ```powershell
 Get-Module -Name Compile-SourceScript -ListAvailable -Verbose
 ```
+
+## FAQ
+
+### Common issues
+
+1. `bash: /path/to/scripting/amxxpc: No such file or directory`
+
+    Install [dependencies](test/scripts/dep/linux/sourcepawn-dependencies.sh) for the compiler.
+
+2. `compiler failed to instantiate: amxxpc32.so: cannot open shared object file: No such file or directory`
```

Comment:
lgtm. Using numbered lists makes the section appear like a sequence of steps to resolve something. Unless of course the reader considered each number as a separate issue.

Reply:
we’ll have them be bulleted then

Reply:
lgtm now @leojonathanoh